src/app/extensions/captcha/shared/captcha-v3/captcha-v3.component.ts

```diff
@@ -41,7 +41,18 @@ export class CaptchaV3Component implements OnInit, OnDestroy {
           take(1),
           switchMap(() =>
             timer(0, 2 * (60 - 3) * 1000).pipe(
-              switchMap(() => this.recaptchaV3Service.execute(this.parentForm.get('captchaAction').value))
+              switchMap(() =>
+                this.recaptchaV3Service.execute(this.parentForm.get('captchaAction').value).pipe(
```

Comment:
use a helper method to have the code only once

Reply:
resolved